website/sidebars.js

```diff
@@ -131,13 +136,12 @@ const sidebarSettings = {
             {
               type: "category",
               label: "Secure your tenant",
+              link: { type: "doc", id: "docs/cloud/secure/ip-restrictions" },
               items: [
+                "docs/cloud/secure/ip-restrictions",
                 "docs/cloud/secure/about-privatelink",
                 "docs/cloud/secure/snowflake-privatelink",
                 "docs/cloud/secure/databricks-privatelink",
-                "docs/cloud/secure/redshift-privatelink",
```

Comment:
is there a reason why are we removing redshift and postgres? we just recently [added](https://dbt-labs.slack.com/archives/C03U5QJEWLC/p1695928007151459?thread_ts=1695202527.175509&cid=C03U5QJEWLC) those per user need

Reply:
![Screenshot 2023-10-09 at 11 25 52](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/0da3c616-060e-4bce-adf2-42d346646031)

Reply:
Added those erroneously removed links back